packages/mobile/src/app/selectors.ts

```diff
@@ -180,29 +182,57 @@ export const registrationStepsSelector = createSelector(
     biometryEnabledSelector,
     activeScreenSelector,
     recoveringFromStoreWipeSelector,
+    skipVerificationSelector,
   ],
-  (chooseRestoreAccount, biometryEnabled, activeScreen, recoveringFromStoreWipe) => {
+  (
+    chooseRestoreAccount,
+    biometryEnabled,
+    activeScreen,
+    recoveringFromStoreWipe,
+    skipVerification
+  ) => {
+    let steps
+    let totalSteps
+    let step
     if (recoveringFromStoreWipe) {
-      return {
-        step: storeWipeRecoverySteps[activeScreen as StoreWipeRecoveryScreens],
-        totalSteps: 3,
-      }
+      steps = storeWipeRecoverySteps
+      totalSteps = 3
```

Comment:
It does happens to be the same since verification has 2 screens in the same step. I was hesitant to code that directly in b/c i'm not sure how stable that will be over time